room_assistance/indico_room_assistance/notifications.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2019 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+import dateutil.parser
+
+from indico.core.config import config
+from indico.core.notifications import make_email, send_email
+from indico.core.plugins import get_plugin_template_module
+from indico.modules.events.requests.models.requests import RequestState
+from indico.modules.rb import rb_settings
+
+
+def _get_email_subject(request, subject):
+    event = request.event
+    return '{prefix}[{room}] {subject}'.format(
+        prefix='[Support Request]',
+        room=event.room.full_name,
+        subject=subject
+    ).strip()
+
+
+def send_email_to_assistance(request, template_name, **template_params):
+    to_list = rb_settings.get('assistance_emails')
```

Comment:
This should be moved to the plugin settings.

Reply:
Good catch! I should just remove it and use the setting from the plugin, because I already have a setting for it.